dev/bin/parallel-lint

```diff
@@ -0,0 +1 @@
+../../htdocs/includes/jakub-onderka/php-parallel-lint/parallel-lint
```

Comment:
What is goal of having a file de/bin/parallel-lint with one line the point to runtime libraries dir of dolibarr ?